Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG base_image=ghcr.io/alphagov/govuk-ruby-base:2.7.6
+ARG builder_image=ghcr.io/alphagov/govuk-ruby-builder:2.7.6
+ 
+FROM $builder_image AS builder
+
+WORKDIR /app
+
+COPY Gemfile* .ruby-version /app/
+
+RUN apt update && apt install libpq-dev -y && \
```

Comment:
We should probably just include libpq-dev in the builder image, but this is fine for now. Maybe add a TODO?

Reply:
I added it